scripts/utility/print_pointnc2ascii.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env python3
+
+'''
+Created on Oct 14, 2022
+
+@author: hsoh
+
+The script reads the point observation NetCDF which is generated by MET tools and prints out the text.
+The header which begins with "#" is added. This can be diabled by adding "--hide-header" option
+
+'''
+
+from os import path
+from optparse import OptionParser
+
+import numpy as np
+from netCDF4 import Dataset
+
+def usage():
+    print(f'Usage: python3 print_nc2ascii.py MET_point_obs_nc <--hide-header> <--use-comma>')
+    print(f'            MET_point_obs_nc: NetCDF filename to read (required)')
+    print(f'                --add-header: to add the header (optional, default: not included)')
+    print(f'                 --use-comma: use the "comma" as separator for comma separated output (optional, default: use spaces)"')
```

Comment:
@hsoh-u can you update the `usage()` here to reflect the option for the outfile options? Thanks.

Reply:
Thank you for catching this. It's updated.

The python way to get the help message:
```
python3 print_pointnc2ascii.py -h
```

Reply:
Thanks Howard. I think the "-h" is optional, is that right? So:
```
python3 print_pointnc2ascii.py -h
```

is the same as:
```
python3 print_pointnc2ascii.py
```
?

Reply:
No, they are different
scripts/utility/print_pointnc2ascii.py  -h
```
Usage: print_pointnc2ascii.py [options]

Options:
  -h, --help            show this help message and exit
  --add-header, --add_header
                         Add header (default: not include header)
  --use-comma, --use_comma
                         Use comma as separator (default: False)
  -o OUT_FILE, --out=OUT_FILE
                         Save the text into the file (default: False)
```

scripts/utility/print_pointnc2ascii.py
```
 == INFO == Missing input NetCDF filename
Usage: python3 print_nc2ascii.py MET_point_obs_nc <--hide-header> <--use-comma> <--out=out_filename>
            MET_point_obs_nc: NetCDF filename to read (required)
                --add-header: to add the header (optional, default: not included)
                 --use-comma: use the "comma" as separator for comma separated output (optional, default: use spaces)"
          --out=out_filename: save the text into the file (optional, default: display to the screen)"
       Note: <> indicates optional arguments
```

Reply:
Nevermind, I see the difference. I will recommend "-h" in the docs.